mk/config.mk

```diff
@@ -206,10 +206,22 @@ CFG_WITH_USER_TA ?= y
 # $(ta-targets).
 
 # Load user TAs from the REE filesystem via tee-supplicant
-# There is currently no other alternative, but you may want to disable this in
-# case you implement your own TA store
 CFG_REE_FS_TA ?= y
 
+# Authentication of TA binaries loaded from the REE filesystem
+#
+# - If CFG_REE_FS_TA_BUFFERED=y: loading TA binary into a temporary buffer,
+#   check the signature, then process the file only if it is valid. This is the
+#   most secure setting.
+# - When disabled: hash the binaries as they are being processed and verify the
+#   signature as a last step. Requires less memory than the first option, but
+#   is less secure because an unsigned malicious binary could trigger potential
```

Comment:
Spontaneously it feels like we shouldn't have something in the code that we know can lead to security issues. This is another way of saying to hacker "look here, we have a potential security hole here". CFG_REE_FS_TA_BUFFERED cannot be the default because of pager? Is that the current conclusion?